ar_exercise/exercise.rb

```diff
@@ -29,20 +29,38 @@ class Apartment < ActiveRecord::Base
 
 # Hint, the following methods will help: `where`, `all`, `find`, `find_by`
 
-# Use Active record to do the following, and store the results **in a variable**
-# example: get every tenant in the DB
-all_tenants = Tenant.all
-
-# get the first tenant in the DB
-# get all tenants older than 65
-# get all apartments whose price is greater than $2300
-# get the apartment with the address "6005 Damien Corners"
-# get all tenants in that apartment
-
-# Use `each` and `puts` to:
-# Display the name and ID # of every tenant
-# Iterate over each apartment, for each apartment, display it's address and rent price
-# Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+# # Use Active record to do the following, and store the results **in a variable**
+# # example: get every tenant in the DB
+# all_tenants = Tenant.all
+#
+# # get the first tenant in the DB
+# first_tenant = Tenant.first
+#
+# # get all tenants older than 65
+# older_tenants = Tenant.where("age > ?", 65)
+#
+# # get all apartments whose price is greater than $2300
+# expensive_apartments = Apartment.where("monthly_rent > ?", 2300)
+#
+# # get the apartment with the address "6005 Damien Corners"
+# damien_apartment = Apartment.find_by(address: "6005 Damien Corners")
+#
+# # get all tenants in that apartment
+# damien_tenants = damien_apartment.tenants
+#
+# # Use `each` and `puts` to:
+# # Display the name and ID # of every tenant
+# # all_tenants.each {|tenant| puts tenant.id.to_s + ' ' + tenant.name}
```

Comment:
This is totally fine, but the 'most idiomatic' style here would be to use string interpolation to avoid having to call `.to_s`.